docs/pages/guides/sharing-before-release.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Sharing before release
```

Comment:
i think this title is a bit unclear

Reply:
Do you have a suggestion?  The original title was "Sharing your app before you ship it to the stores" but that's pretty long

https://linear.app/expo/issue/WR-61/sharing-your-app-before-you-ship-it-to-the-stores

Reply:
i think it's unclear because:
- "before release" could be read as "before the first time you submit to the store" 
- "Sharing" is not a word i think people associate with distributing test builds, it is strongly associated with social media sharing

maybe: "Creating and distributing test builds" or something